fmn/delivery/backends/mail.py

```diff
@@ -0,0 +1,131 @@
+from .base import BaseBackend
+
+from kitchen.text.converters import to_bytes
+
+import email
+import logging
+
+from twisted.internet import defer
+from twisted.mail import smtp
+
+from fmn.lib import models
+from fmn.util import get_fas_email
```

Comment:
I recommend re-grouping these imports for PEP-8 (std lib, third party, and fmn).